### PR TITLE
Improve chinese translation

### PR DESCRIPTION
**Pull request by liuzhenji**

### CODE REVIEW

File: content/zh/docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Kubernetes also stores its serialized state (currently in [etcd](https://coreos.
 
 Kubernetes itself is decomposed into multiple components, which interact through its API.
 -->
-[API协议文档](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md)描述了主系统和API概念。
+[API协议文档](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md)记录了 k8s API的所有协议规范。
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/kubernetes-api.md]:
```suggestion
[API 约定文档](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md) 描述了 k8s API 的总体约定。
```

Comment (tengqm) [reply content/zh/docs/concepts/overview/kubernetes-api.md]: convention 在这个上下文中还是一种约定俗成的东西，还谈不上协议。具体到 API Specification 的时候才谈得上协议。供参考。

Comment (liuzhenji) [reply content/zh/docs/concepts/overview/kubernetes-api.md]: 嗯是的，中括号的原解释也有不妥。

Comment (tanjunchen) [reply content/zh/docs/concepts/overview/kubernetes-api.md:25]: @liuzhenji 
any update?

**Comment by tengqm** [comment]
Lemme try close and reopen to trigger a build.
/close

**Comment by tengqm** [comment]
@liuzhenji can you help rebase this onto master to trigger a rebuild?

**Comment by liuzhenji** [comment]
> @liuzhenji can you help rebase this onto master to trigger a rebuild?

sure

**Comment by liuzhenji** [comment]
> @liuzhenji can you help rebase this onto master to trigger a rebuild?

but how do i do ,re-PR?

**Comment by tengqm** [comment]
On you local machine:

```
git checkout master
git fetch upstream
git rebase upstream/master
git push

git checkout patch-2
git rebase master
git commit --amend
git push -f
```

**Comment by liuzhenji** [comment]
> On you local machine:
> 
> ```
> git checkout master
> git fetch upstream
> git rebase upstream/master
> git push
> 
> git checkout patch-2
> git rebase master
> git commit --amend
> git push -f
> ```

already failed a lot of times tring git clone. close this PR temp.

**Pull request closed without merging**